### ISSUE TITLE
Linter fail when method has return type declaration

### ISSUE THREAD

**Issue by michalzielanski** [FriendsOfPHP/PHP-CS-Fixer#2542]
PHP version: 5.6.30
PHP CS Fixer version: 2.1.0

Command:
```
php-cs-fixer fix test.php --diff --verbose
```

Output:
```
Loaded config default.
I
Legend: ?-unknown, I-invalid file syntax, file ignored, S-Skipped, .-no changes, F-fixed, E-error

Fixed all files in 1.167 seconds, 9.250 MB memory used

Files that were not fixed due to errors reported during linting before fixing:
   1) /(...)/test.php
```

Test file:
```php
<?php

abstract class Test
{
    protected $prop;

    public function getProp() : string
    {
        return $this->prop;
    }
}
```

**Comment by SpacePossum** [comment]
The code you try to fix is not valid for PHP 5.6. If you want to fix PHP 7 code you must run the tool of at least that version.
This cannot be helped (easly) as we use the build in PHP linter and tokenizer.

**Comment by michalzielanski** [comment]
Ok. With 7.0.15 works properly.
